.env.example

```diff
@@ -1,3 +1,4 @@
 NASA_API_KEY=
 NASA_API_URL=https://api.nasa.gov/neo/rest/v1/feed
+ROVER_API_URL=https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos
```

Comment:
Do you think it might be worth separating the base URL and endpoints?

Reply:
and it seems to me that it’s not worth storing endpoints in environment variables
I would leave only the base URL in env, and leave the rest in the config

Reply:
Probably yes

Reply:
I could also add interceptor to avoid repeating base url in config, what do you think?